src/participant/participant.ts

```diff
@@ -130,9 +134,51 @@ export default class ParticipantController {
    * in the chat. To work around this, we can write a message as the user, which will
    * trigger the chat handler and give us access to the model.
    */
-  writeChatMessageAsUser(message: string): Thenable<unknown> {
+  async sendMessageToParticipant(
```

Comment:
Could you maybe also to refactor this part to use the sendMessageToParticipant helper?

https://github.com/mongodb-js/vscode/blob/main/src/mdbExtensionController.ts#L952-L959